<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountCreate</class>
 <widget class="QDialog" name="AccountCreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AccountCreate</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
 font-family:century gothic;
 font-size:14px;
}

#AccountCreate
{
 background:url(:/LoginUI/Singup_Background.png);
}
</string>
  </property>
  <widget class="QLineEdit" name="Edit_Name">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>330</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
 background:transparent;
 border:none;
 font-size:24px;
 color:#979797;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_Password">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>225</y>
     <width>330</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
 background:transparent;
 border:none;
 font-size:24px;
 color:#979797;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Comment">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background:transparent;
	color:#952e25;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Create_Button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>121</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background:url(&quot;:/LoginUI/Signup_Accept.png&quot;);
	border:none;
}

QPushButton:hover
{
	background:url(&quot;:/LoginUI/Signup_Accept_Over.png&quot;);
	border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancle_Button">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>121</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background:url(&quot;:/LoginUI/Signup_Cancle.png&quot;);
	border:none;
}

QPushButton:hover
{
	background:url(&quot;:/LoginUI/Signup_Cancle_Over.png&quot;);
	border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resource/LoginUI.qrc"/>
 </resources>
 <connections/>
</ui>
